<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditGearClass</class>
 <widget class="QWidget" name="EditGearClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit equipment[*]</string>
  </property>
  <property name="windowIcon">
   <iconset resource="analogexif.qrc">
    <normaloff>:/images/icons/camera_edit.png</normaloff>:/images/icons/camera_edit.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>100</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Equipment</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="30,70">
      <item>
       <layout class="QGridLayout" name="gridLayout" rowstretch="0,30,0,30,0,15,0,15">
        <property name="horizontalSpacing">
         <number>0</number>
        </property>
        <property name="verticalSpacing">
         <number>2</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Cameras and lens</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="5">
         <widget class="GearTreeView" name="gearView">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>60</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="statusTip">
           <string>Select the equipment to edit</string>
          </property>
          <property name="styleSheet">
           <string notr="true">/* keep selection even when focus is lost */
QTreeView::item:selected:!active{
	selection-color:						palette(highlighted-text);
	selection-background-color:	palette(highlight);
}</string>
          </property>
          <property name="dragEnabled">
           <bool>true</bool>
          </property>
          <property name="dragDropOverwriteMode">
           <bool>false</bool>
          </property>
          <property name="dragDropMode">
           <enum>QAbstractItemView::DragDrop</enum>
          </property>
          <property name="defaultDropAction">
           <enum>Qt::MoveAction</enum>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
          <property name="uniformRowHeights">
           <bool>true</bool>
          </property>
          <property name="itemsExpandable">
           <bool>true</bool>
          </property>
          <attribute name="headerVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="headerStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="headerVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="headerStretchLastSection">
           <bool>true</bool>
          </attribute>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Films</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="5">
         <widget class="GearListView" name="filmView">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>40</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="statusTip">
           <string>Select the film to edit</string>
          </property>
          <property name="styleSheet">
           <string notr="true">/* keep selection even when focus is lost */
QListView::item:selected:!active{
	selection-color:						palette(highlighted-text);
	selection-background-color:	palette(highlight);
}</string>
          </property>
          <property name="dragEnabled">
           <bool>true</bool>
          </property>
          <property name="dragDropMode">
           <enum>QAbstractItemView::DragDrop</enum>
          </property>
          <property name="defaultDropAction">
           <enum>Qt::MoveAction</enum>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Developers</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0" colspan="5">
         <widget class="GearListView" name="developerView">
          <property name="font">
           <font>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="statusTip">
           <string>Select the developer to edit</string>
          </property>
          <property name="styleSheet">
           <string notr="true">/* keep selection even when focus is lost */
QListView::item:selected:!active{
	selection-color:						palette(highlighted-text);
	selection-background-color:	palette(highlight);
}</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Authors</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0" colspan="5">
         <widget class="GearListView" name="authorView">
          <property name="font">
           <font>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="statusTip">
           <string>Select the author to edit</string>
          </property>
          <property name="styleSheet">
           <string notr="true">/* keep selection even when focus is lost */
QListView::item:selected:!active{
	selection-color:						palette(highlighted-text);
	selection-background-color:	palette(highlight);
}</string>
          </property>
         </widget>
        </item>
        <item row="6" column="4">
         <widget class="QToolButton" name="delAuthorBtn">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Delete selected author(s)</string>
          </property>
          <property name="statusTip">
           <string>Delete selected author(s)</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="analogexif.qrc">
            <normaloff>:/images/icons/delete.png</normaloff>:/images/icons/delete.png</iconset>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="QToolButton" name="addAuthorBtn">
          <property name="toolTip">
           <string>Add new author</string>
          </property>
          <property name="statusTip">
           <string>Add new author</string>
          </property>
          <property name="text">
           <string>Add</string>
          </property>
          <property name="icon">
           <iconset resource="analogexif.qrc">
            <normaloff>:/images/icons/add.png</normaloff>:/images/icons/add.png</iconset>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QToolButton" name="addDevBtn">
          <property name="toolTip">
           <string>Add new developer</string>
          </property>
          <property name="statusTip">
           <string>Add new developer</string>
          </property>
          <property name="text">
           <string>Add</string>
          </property>
          <property name="icon">
           <iconset resource="analogexif.qrc">
            <normaloff>:/images/icons/add.png</normaloff>:/images/icons/add.png</iconset>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="4">
         <widget class="QToolButton" name="delDevBtn">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Delete selected developer(s)</string>
          </property>
          <property name="statusTip">
           <string>Delete selected developer(s)</string>
          </property>
          <property name="text">
           <string>Delete</string>
          </property>
          <property name="icon">
           <iconset resource="analogexif.qrc">
            <normaloff>:/images/icons/delete.png</normaloff>:/images/icons/delete.png</iconset>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QToolButton" name="addFilmBtn">
          <property name="toolTip">
           <string>Add new film</string>
          </property>
          <property name="statusTip">
           <string>Add new film</string>
          </property>
          <property name="text">
           <string>Add</string>
          </property>
          <property name="icon">
           <iconset resource="analogexif.qrc">
            <normaloff>:/images/icons/add.png</normaloff>:/images/icons/add.png</iconset>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QToolButton" name="delFilmBtn">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Delete selected film(s)</string>
          </property>
          <property name="statusTip">
           <string>Delete selected film(s)</string>
          </property>
          <property name="text">
           <string>Delete</string>
          </property>
          <property name="icon">
           <iconset resource="analogexif.qrc">
            <normaloff>:/images/icons/delete.png</normaloff>:/images/icons/delete.png</iconset>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QToolButton" name="addLensBtn">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Add new lens</string>
          </property>
          <property name="statusTip">
           <string>Add new lens</string>
          </property>
          <property name="text">
           <string>Add</string>
          </property>
          <property name="icon">
           <iconset resource="analogexif.qrc">
            <normaloff>:/images/icons/add.png</normaloff>:/images/icons/add.png</iconset>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QToolButton" name="delGearBtn">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Delete selected equipment</string>
          </property>
          <property name="statusTip">
           <string>Delete selected equipment</string>
          </property>
          <property name="text">
           <string>Delete</string>
          </property>
          <property name="icon">
           <iconset resource="analogexif.qrc">
            <normaloff>:/images/icons/delete.png</normaloff>:/images/icons/delete.png</iconset>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QToolButton" name="addCameraBtn">
          <property name="toolTip">
           <string>Add new camera body</string>
          </property>
          <property name="statusTip">
           <string>Add new camera body</string>
          </property>
          <property name="text">
           <string>Add camera</string>
          </property>
          <property name="icon">
           <iconset resource="analogexif.qrc">
            <normaloff>:/images/icons/camera_add.png</normaloff>:/images/icons/camera_add.png</iconset>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QToolButton" name="dupGearBtn">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Duplicate selected equipment</string>
          </property>
          <property name="statusTip">
           <string>Duplicate selected equipment</string>
          </property>
          <property name="text">
           <string>Duplicate</string>
          </property>
          <property name="icon">
           <iconset resource="analogexif.qrc">
            <normaloff>:/images/icons/page_copy.png</normaloff>:/images/icons/page_copy.png</iconset>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QToolButton" name="dupFilmBtn">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Duplicate selected film(s)</string>
          </property>
          <property name="statusTip">
           <string>Duplicate selected film(s)</string>
          </property>
          <property name="text">
           <string>Duplicate</string>
          </property>
          <property name="icon">
           <iconset resource="analogexif.qrc">
            <normaloff>:/images/icons/page_copy.png</normaloff>:/images/icons/page_copy.png</iconset>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QToolButton" name="dupDevBtn">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Duplicate selected developer(s)</string>
          </property>
          <property name="statusTip">
           <string>Duplicate selected developer(s)</string>
          </property>
          <property name="text">
           <string>Duplicate</string>
          </property>
          <property name="icon">
           <iconset resource="analogexif.qrc">
            <normaloff>:/images/icons/page_copy.png</normaloff>:/images/icons/page_copy.png</iconset>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="6" column="3">
         <widget class="QToolButton" name="dupAuthorBtn">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Duplicate selected author(s)</string>
          </property>
          <property name="statusTip">
           <string>Duplicate selected author(s)</string>
          </property>
          <property name="text">
           <string>Duplicate</string>
          </property>
          <property name="icon">
           <iconset resource="analogexif.qrc">
            <normaloff>:/images/icons/page_copy.png</normaloff>:/images/icons/page_copy.png</iconset>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
          <property name="arrowType">
           <enum>Qt::NoArrow</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,0,0,0">
        <property name="horizontalSpacing">
         <number>0</number>
        </property>
        <property name="verticalSpacing">
         <number>2</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Properties</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="4">
         <widget class="GearTableView" name="metadataView">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="showDropIndicator" stdset="0">
           <bool>false</bool>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="showGrid">
           <bool>false</bool>
          </property>
          <property name="gridStyle">
           <enum>Qt::SolidLine</enum>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>150</number>
          </attribute>
          <attribute name="horizontalHeaderHighlightSections">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderMinimumSectionSize">
           <number>150</number>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderCascadingSectionResizes">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>13</number>
          </attribute>
          <attribute name="verticalHeaderMinimumSectionSize">
           <number>13</number>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>13</number>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderCascadingSectionResizes">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderMinimumSectionSize">
           <number>150</number>
          </attribute>
          <attribute name="verticalHeaderMinimumSectionSize">
           <number>13</number>
          </attribute>
          <attribute name="horizontalHeaderHighlightSections">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>150</number>
          </attribute>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QToolButton" name="delTagBtn">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Delete meta tag</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="analogexif.qrc">
            <normaloff>:/images/icons/delete.png</normaloff>:/images/icons/delete.png</iconset>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QToolButton" name="addTagBtn">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Add new meta tag</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="analogexif.qrc">
            <normaloff>:/images/icons/add.png</normaloff>:/images/icons/add.png</iconset>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionAdd_new_camera_body">
   <property name="text">
    <string>Add new camera body</string>
   </property>
  </action>
  <action name="actionAdd_new_camera_lens">
   <property name="text">
    <string>Add new camera lens</string>
   </property>
  </action>
  <action name="actionAdd_new_film">
   <property name="text">
    <string>Add new film</string>
   </property>
  </action>
  <action name="actionDuplicate">
   <property name="text">
    <string>Duplicate...</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionAdd_new_developer">
   <property name="text">
    <string>Add new developer</string>
   </property>
  </action>
  <action name="actionAdd_new_author">
   <property name="text">
    <string>Add new author</string>
   </property>
  </action>
  <action name="actionAdd_new_meta_tag">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Add new meta tag</string>
   </property>
  </action>
  <action name="actionDelete_meta_tag">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Delete meta tag</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GearListView</class>
   <extends>QListView</extends>
   <header>gearlistview.h</header>
  </customwidget>
  <customwidget>
   <class>GearTreeView</class>
   <extends>QTreeView</extends>
   <header>geartreeview.h</header>
  </customwidget>
  <customwidget>
   <class>GearTableView</class>
   <extends>QTableView</extends>
   <header>geartableview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>gearView</tabstop>
  <tabstop>filmView</tabstop>
  <tabstop>developerView</tabstop>
  <tabstop>authorView</tabstop>
  <tabstop>metadataView</tabstop>
  <tabstop>addCameraBtn</tabstop>
  <tabstop>addLensBtn</tabstop>
  <tabstop>dupGearBtn</tabstop>
  <tabstop>delGearBtn</tabstop>
  <tabstop>addFilmBtn</tabstop>
  <tabstop>dupFilmBtn</tabstop>
  <tabstop>delFilmBtn</tabstop>
  <tabstop>addDevBtn</tabstop>
  <tabstop>dupDevBtn</tabstop>
  <tabstop>delDevBtn</tabstop>
  <tabstop>addAuthorBtn</tabstop>
  <tabstop>dupAuthorBtn</tabstop>
  <tabstop>delAuthorBtn</tabstop>
  <tabstop>addTagBtn</tabstop>
  <tabstop>delTagBtn</tabstop>
 </tabstops>
 <resources>
  <include location="analogexif.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>addFilmBtn</sender>
   <signal>clicked()</signal>
   <receiver>actionAdd_new_film</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addDevBtn</sender>
   <signal>clicked()</signal>
   <receiver>actionAdd_new_developer</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>340</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addAuthorBtn</sender>
   <signal>clicked()</signal>
   <receiver>actionAdd_new_author</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>427</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addCameraBtn</sender>
   <signal>clicked()</signal>
   <receiver>actionAdd_new_camera_body</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>166</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>delTagBtn</sender>
   <signal>clicked()</signal>
   <receiver>actionDelete_meta_tag</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>697</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
